types/fs-base-ui/html_ui/JS/SimPlane.d.ts

```diff
@@ -0,0 +1,421 @@
+        function getAutoPilotGlideslopeHold(): boolean | null;
+        function getAutoPilotApproachType(): ApproachType | null;
+        function getAutoPilotApproachLoaded(): boolean | null;
+        function getAutoPilotNavAidState(user: number, _switch: any): number | null;
+        function getAutoPilotIsHeadingAligned(): boolean | null;
+        function getNextWaypointName(): string | null;
+        function getNextWaypointTrack(): Degrees | null;
+        function getNextWaypointDistance(): NauticalMiles | null;
+        function getNextWaypointETA(): Seconds | null;
+        function getFlightTime(): Seconds | null;
+        function getCurrentUTC(): Seconds | null;
+        function getEngineCount(): number | null;
+        function getEngineActive(engineIndex: number): boolean | null;
+        function getEngineThrottle(engineIndex: number): Percent | null;
+        function getEngineThrottleMode(engineIndex: number): ThrottleMode | null;
+        function getEngineCommandedN1(engineIndex: number): Percent | null;
+        function getEngineThrottleCommandedN1(engineIndex: number): Percent | null;
+        function getEngineThrottleMaxThrust(engineIndex: number): number;
+        function getEngineThrustTakeOffMode(engineIndex: number): number | null;
+        function getEngineThrustClimbMode(engineIndex: number): number | null;
+        /**
+         * Equal to getEngineThrottle
+         */
+        function getAutopilotThrottle(engineIndex: number): Percent | null;
+        /**
+         * Equal to getEngineCommandedN1
+         */
+        function getAutopilotCommandedN1(engineIndex: number): Percent | null;
+        function getEngineType(): EngineType | null;
+        function getEngineRPM(engineIndex: number): RotationsPerMinute | null;
+        function getEnginePower(engineIndex: number): Percent | null;
+        function getMinCruiseRPM(): RotationsPerMinute | null;
+        function getMaxCruiseRPM(): RotationsPerMinute | null;
+        function getMaxIndicatedRPM(): RotationsPerMinute | null;
+        function getMaxRatedRPM(): RotationsPerMinute | null;
+        function getPropellerType(): PropellerType | null;
+        function getNbPropellers(): number | null;
+        function getInclinometer(): Position | null;
+        function getAngleOfAttack(): Angl16 | null;
+        function getOrientationAxis(): XYZ | null;
+        function getAltitude(): Feet | null;
+        function getGroundReference(): Feet | null;
+        function getTurnRate(): RadiansPerSecond | null;
+        function getHeadingMagnetic(): Heading | null;
+        function getPitch(): Degrees | null;
+        function getBank(): Degrees | null;
+        function getFlightDirectorPitch(): Degrees | null;
+        function getFlightDirectorBank(): Degrees | null;
+        function getIsGrounded(): boolean;
+        function getAltitudeAboveGround(forceSimVarCall?: boolean): Feet;
+        function getCrossoverAltitude(cas, mach: Mach): Feet | null;
```

Comment:
`cas` has no type.

Reply:
Using the JS code, I was unable to determine the type. Any suggestions?

Reply:
I'd guess that `cas` would be the calibrated airspeed, which would be measured in Knots.